app/src/main/java/com/greenfox/peridot/peridot_coz_android/activity/MainActivity.java

```diff
@@ -44,8 +55,22 @@ public void onResponse(Call<User> call, Response<User> response) {
             }
             @Override
             public void onFailure(Call<User> call, Throwable t) {
-                }});
-        logoutButton = (Button) findViewById(R.id.logoutButton);
+            }});
+
+
+        DrawerLayout drawer = (DrawerLayout) findViewById(R.id.drawer_layout);
+        ActionBarDrawerToggle toggle = new ActionBarDrawerToggle(
+                this, drawer, myToolbar, R.string.navigation_drawer_open, R.string.navigation_drawer_close);
+        drawer.setDrawerListener(toggle);
+        toggle.syncState();
+
+        NavigationView navigationView = (NavigationView) findViewById(R.id.nav_view);
+        navigationView.setNavigationItemSelectedListener(this);
+
+        KingdomOverviewFragment kingdomOverviewFragment= new KingdomOverviewFragment();
+        fragmentTransaction = getSupportFragmentManager().beginTransaction();
+        fragmentTransaction.replace(R.id.content_frame, kingdomOverviewFragment);
+        fragmentTransaction.commit();
```

Comment:
@radaladar `FragmentTransaction` is chainable, you don't need to store it in a variable, you can just call the methods on the previous ones